config/betterquesting/DefaultQuests/Quests/GettingAround-bAal8kZ3R8eRTwaHgbYkKA==/Teleporter-tJdwTeS6TOOFACQhNuUvEw==.json

```diff
@@ -0,0 +1,62 @@
+{
+  "questIDHigh:4": -5433750945547006749,
+  "preRequisites:9": {
+    "0:10": {
+      "questIDHigh:4": 0,
+      "questIDLow:4": 176
+    }
+  },
+  "questIDLow:4": -8863044341591625965,
+  "properties:10": {
+    "betterquesting:10": {
+      "snd_complete:8": "random.levelup",
+      "taskLogic:8": "AND",
+      "visibility:8": "NORMAL",
+      "isMain:1": 0,
+      "simultaneous:1": 0,
+      "icon:10": {
+        "id:8": "IC2:blockMachine2",
+        "Count:3": 1,
+        "Damage:2": 0,
+        "OreDict:8": ""
+      },
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "Teleporter",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "IndustrialCraft2 Teleporter allows for teleportation between 2 connected teleporters.\nRequires energy storing block right next to it due to the massive power demand.\n\n\n§3Energy required is determined by the distance between the teleporters and amount of items in the inventory.\nThe teleporter is quite buggy according to the offcial wiki."
```

Comment:
This also has a bunch of articles missing. 'The IndustrialCraft2 Teleporter...', 'It requires an energy storing block...', 'The energy required ...', 'the amount of items'

Reply:
english is hard :fail: